install-build-deps.sh

```diff
@@ -73,3 +73,7 @@ if [ -e /usr/bin/pacman ] ; then
       sudo pacman -S --needed $TO_INSTALL
   fi
 fi
+
+# Install latest CMake
+wget -nv https://github.com/TheAssassin/CMake/releases/download/continuous/cmake-continuous-"$ARCH".AppImage -O /usr/bin/cmake
```

Comment:
Do we really need not only a newer CMake than what is available on CentOS 6, but even a __private__ one? Why can't we at least use an official one? If we must keep a private one, let's move that repo to the AppImage organization at least, so that we don't depend on the personal repos of a key contributor ;-)

Reply:
No, we don't need a private one, another legacy issue, thanks for spotting this.